looprpc/client.proto

```diff
@@ -127,6 +127,12 @@ message LoopOutRequest {
     lowest routing fee for the swap payment to the server.
     */
     uint64 loop_out_channel = 8;
+
+    /**
+    The number of blocks from the on-chain HTLC's confirmation height that it
+    should be swept within.
+    */
+    int32 sweep_conf_target = 9;
```

Comment:
Having both `max_miner_fee` and `sweep_conf_target` somehow suggests that we have not yet settled on the best set of parameters to control the chain fees. But I think it is something we can find out later.

There is also the following case to be aware of: if we get really close to the expiry, we may actually want to spend an amount on chain fees up to the full swap amount, because we are also risking losing the full amount if we don't make it in time.

Reply:
Yeah I'm not sure how useful `max_miner_fee` is anymore given the risk once the preimage is revealed. Seems like we'd want a better strategy at that point, e.g., positive yield?